patches/server/0936-Uncompressed-Regions.patch

```diff
@@ -0,0 +1,48 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Peter Crawley <peterzcrawley@gmail.com>
+Date: Tue, 25 Oct 2022 17:38:53 +0100
+Subject: [PATCH] Uncompressed Regions
+
+
+diff --git a/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java b/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
+index 456595e4b7e0c7f50617aa2694b0d2dfc368ab81..25fa77cb02c8cd6498a3df129598e3798f21a202 100644
+--- a/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
++++ b/src/main/java/io/papermc/paper/configuration/GlobalConfiguration.java
+@@ -130,6 +130,7 @@ public class GlobalConfiguration extends ConfigurationPart {
+         public double playerMaxConcurrentLoads = 20.0;
+         public double globalMaxConcurrentLoads = 500.0;
+         public double playerMaxChunkLoadRate = -1.0;
++        public boolean saveRegionsUncompressed = false;
+     }
+ 
+     public UnsupportedSettings unsupportedSettings;
```

Comment:
Config changes go in the config patch now

Reply:
Moved to config patch